lib/bozo/test_runners/nunit3.rb

```diff
@@ -57,6 +61,10 @@ def runner_args
         args << '--x86'
       end
 
+      unless @agents.blank?
+        args << '--agents=#{@agents}'
```

Comment:
You've not used the correct string here you need to use double quotes to get string interpolation